src/components/forms/TextField/TextField.js

```diff
@@ -0,0 +1,47 @@
+import {
+  FormFieldDescription,
+  FormLabel,
+  Paragraph,
+  Textbox,
+  FormField as UtrechtFormField,
+} from '@utrecht/component-library-react';
+import classNames from 'classnames';
+import React, {useRef} from 'react';
+
+import {getBEMClassName} from '../../../utils';
+
+export const TextField = ({
+  label = '',
+  isRequired = false,
+  description = '',
+  id = '',
+  disabled,
+  invalid,
+}) => {
+  const labelClassName = classNames(getBEMClassName('label'), {'field-required': isRequired});
```

Comment:
This `field-required` is because of the existing/legacy styles, right? I think I'd rather normalize that to be a modifier:

```suggestion
  const labelClassName = getBEMClassName(
    'label',
    [isRequired && 'required'].filter(Boolean),
  ));
```
and then update the `_label.scss` file to have:

```scss
@use 'microscope-sass/lib/bem';

// ... other imports

.#{prefix(label)} {
  @include body;
  @include body--big;
  @include anchor.extend-utrecht-link;
  display: block;
  font-weight: var(--of-label-font-weight);

  // .field-required is deprecated, use the modifier instead.
  &.field-required,
  @include bem.modifier('required') {
    &:after {
      content: '*';
      color: var(--of-color-danger);
      z-index: 0; // otherwise this is above the progress indicator in some browsers
    }  
  }
}
```